### DISCUSSION TITLE
How do I find the listener?

### DISCUSSION THREAD

**Discussion by SabreRunner** [unity-atoms/unity-atoms#450] [Q&A]
I have an event SO and I know I have a listener for it somewhere in the scene. How do I find it quickly?

**Comment by soraphis** [comment]
I used this one in the past and it was working not too bad:
https://forum.unity.com/threads/experimental-dependency-viewer.1158899/

I used this one in the past and can't really complain:
https://github.com/yasirkula/UnityAssetUsageDetector
(also available via the store: https://assetstore.unity.com/packages/tools/utilities/asset-usage-detector-112837 )
